travis-ci/cleanup-integration-tests.sh

```diff
@@ -4,31 +4,37 @@ set -e
 
 . ./travis-ci/set-env-vars.sh
```

Comment:
these require `simple-git` in the select-stack process and need to come after the package installs.

Reply:
Fixed.